pages/programs/youth-managed-farms.css

```diff
@@ -0,0 +1,168 @@
+/* Importing stuff! */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+
+/*Now the actual code */
+
+/* Grid Items */
+.grid-st {
+
+}
+.grid-mid {
+
+}
+.grid-end {
+	grid-column-start:2;
+	grid-column-end: 5;
+}
+/* Header Items */
+
+#headerBR {
+	display: grid;
+  	grid-template-columns: auto auto auto auto;
```

Comment:
A grid with all `auto` values seems like a great opportunity to refactor this code to be more efficient, maybe `1fr` in place of the `auto`s is a great start.